include/uart.h

```diff
@@ -180,7 +185,7 @@ struct uart_event_rx {
 	u8_t *buf;
 	/** @brief Offset from buffer start to currently received data. */
```

Comment:
Imo something like: ` Currently received data offset in bytes.` should be enough. If not then adding ` First byte is located at buf[offset]` should be more than enough.

Reply:
Ok, I have added this to my changes as well.